deploy/docs/additional_prometheus_configuration.md

```diff
@@ -149,70 +149,54 @@ prometheus-operator:  # For values.yaml
 
 ### Create a new HTTP source in Sumo Logic.
 
-To avoid [blacklisting](https://help.sumologic.com/Metrics/Understand_and_Manage_Metric_Volume/Blacklisted_Metrics_Sources) metrics should be distributed across multiple HTTP sources. You can [follow these steps](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/HTTP-Source) to create a new HTTP source for your custom metrics. Make note of the URL as you will need it in the next step.
+To avoid [blacklisting](https://help.sumologic.com/Metrics/Understand_and_Manage_Metric_Volume/Blacklisted_Metrics_Sources) metrics should be distributed across multiple HTTP sources. You can create a new HTTP source using `values.yaml`:
 
-### Update the metrics.conf Fluentd Configuration
-
-Next, you will need to update the Fluentd configuration to ensure Fluentd routes your custom metrics to the HTTP source you created in the previous step.
-
-  * First, base64 encode the HTTP source URL from the previous step by running `echo <HTTP_SOURCE_URL> | base64`.  Replace `<HTTP_SOURCE_URL>` with the URL from step 3.
-  * Next, you can edit the secret that houses all the HTTP sources URLs. Assuming you installed the collector in the  `sumologic` namespace, you can run `kubectl -n sumologic edit secret sumologic` or edit the YAML you deployed when you set up collection.
-  * In the `data` section, add a new key and the base64 encoded value you created. The following is just a snippet of the secret for an example. Do not alter the existing content, you simply want to add a new key.
-  
 ```yaml
-data:
+sumologic:
   # ...
-  my-custom-metrics: <base64EncodedURL>
-  kind: Secret
+  sources:
+    # ...
+    metrics:
+      # ...
+      my_source:
+        name: my-source-name
 ```
 
-  * Next you need to edit the Fluentd Deployment and add a new environment variable, pointing to the new secret.  Assuming you installed the collector in the  `sumologic` namespace, you can run `kubectl -n sumologic edit deployment collection-sumologic` or edit the YAML you deployed when you set up collection. Note, if you installed using helm, the name of the deployment may be different depending on how you installed the helm chart.
-  * Locate the `SUMO_ENDPOINT_LOGS` environment variable in the YAML and add a new environment variable that points to the secret key you created. The following is an example.
-  
+This will create a new HTTP source with the name `my-source-name` in your Sumo Logic account.
+
+**This configuration doesn't modify or remove HTTP sources from your account.
+If you rename a source in `values.yaml`, the new source will be added to your Sumo Logic account**
```

Comment:
we might want to specify that this HTTP source will be added to the same collector, though maybe it's implied? not too sure about this